docs/project/linux-performance-tracing.md

```diff
@@ -172,7 +172,7 @@ To install dotnet symbol issue the command
 ```
      dotnet tool install -g dotnet-symbol
 ```
-With that installed download the symbols to a local directory.  if your installed version of the .NET Core runtime is
+With that installed download the symbols to a local directory.  if your installed version of the .NET runtime is
 2.1.0 the command to do this is
```

Comment:
Should update version

Reply:
This tracing documentation is spread over multiple places. I believe it would be best to delete this doc, and centralize the documentation on the official docs site. I have opened https://github.com/dotnet/diagnostics/issues/933 on it.